lib/vector-vrl/web-playground/README.md

```diff
@@ -13,7 +13,7 @@ To build the project we need to use `wasm-pack`. This compiles our Rust code
 to WebAssembly which can then be used within the browser. Install it by running:
 
 ```shell
-cargo install --version 0.10.3 wasm-pack
+cargo install wasm-pack
```

Comment:
Version `0.10.3` is what is [used in CI](https://github.com/vectordotdev/vector/blob/master/scripts/ensure-wasm-pack-installed.sh#L3). I think there could be some value to using the same version in development. Did you find you had to upgrade? Maybe we should upgrade the version in CI too? Either way installing a specific version seems prudent to avoid issues that only show up locally or in CI.  Could also add a `--locked` here to make sure the same dependency versions the crate was tagged with are used.

Reply:
Good points, thanks! I didn't have to upgrade so we can keep the 0.10.3 version for now. Updated the above to `cargo install --locked --version 0.10.3 wasm-pack`.